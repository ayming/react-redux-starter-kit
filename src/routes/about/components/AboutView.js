import React, { PureComponent } from 'react'

import Link from '../../../components/Link'
import s from './AboutView.scss'

class AboutView extends PureComponent {
  render() {
    return (
      <div className={s.wrapper}>
        <p>
          This starter kit is designed to get you up and running with a bunch of
          awesome front-end technologies.
        </p>
        <Link to="/">HOME</Link> <Link to="/counter">COUNTER</Link>
      </div>
    )
  }
}

export default AboutView
