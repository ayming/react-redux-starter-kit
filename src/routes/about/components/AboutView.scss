.wrapper {
  text-align: center;
  font-size: large;
}
