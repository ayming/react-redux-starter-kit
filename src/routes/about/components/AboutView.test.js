/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import AboutView from './AboutView'

describe('AboutView', () => {
  test('renders correctly', () => {
    const wrapper = renderer.create(<AboutView />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
