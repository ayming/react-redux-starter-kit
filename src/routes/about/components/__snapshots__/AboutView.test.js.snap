// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView renders correctly 1`] = `
<div
  className="wrapper"
>
  <p>
    This starter kit is designed to get you up and running with a bunch of awesome front-end technologies.
  </p>
  <a
    href="/"
    method="push"
    onClick={[Function]}
  >
    HOME
  </a>
   
  <a
    href="/counter"
    method="push"
    onClick={[Function]}
  >
    COUNTER
  </a>
</div>
`;
